app/views/searches/_form.html.erb

```diff
@@ -0,0 +1,14 @@
+<form action="<%= scrapbook_search_path %>" method="GET" id="search-form">
+  <div class="form-group">
+    <label class="form-label" for="query">
+      <span class="heading-small">Work, skills and interests</span>
+    </label>
+    <input type="text" name="query" id="query" class="form-control"
+           title="Work, skills and interests"
+           value="<%= local_assigns[:search_term] %>">
```

Comment:
`local_assigns` is a variable in Rails partials which gives hash-like access to all of the variables passed to the partial. 

Its useful both for optional variables (as in this case), and also for required variables, where you'd use `local_assigns.fetch(:search_term)` to throw an error if the variable hasn't been provided.